app/api/utility.py

```diff
@@ -32,17 +32,31 @@
 
 CATEGORICAL_DOMAINS = [SEX, DIAGNOSIS, IMAGE_MODAL]
 
+# Query choices
+IMAGE_MODAL_CHOICES = {
+    "All": None,
```

Comment:
Do we necessarily need to include `image_modal="All"` as an option if the behaviour is (from what I can tell) the same as when you don't filter for age in your query at all, since this is an optional parameter?

Reply:
I agree. I only kept it in case we need it later in the frontend.